internal/pkg/config/input.go

```diff
@@ -56,21 +58,30 @@ func (c *ServerBulk) InitDefaults() {
 	c.FlushMaxPending = 8
 }
 
+type StaticPolicyTokens struct {
+	// Enabled is a flag to enable static policy tokens
+	Enabled bool `config:"enabled"`
+	// PolicyTokens is a map of policy ID to Enrollment API Key
+	// this token will not be visible in Kibana UI
+	PolicyTokens map[string]string `config:"policy_tokens"`
+}
+
 // Server is the configuration for the server
 type Server struct {
-	Host              string                  `config:"host"`
-	Port              uint16                  `config:"port"`
-	InternalPort      uint16                  `config:"internal_port"`
-	TLS               *tlscommon.ServerConfig `config:"ssl"`
-	Timeouts          ServerTimeouts          `config:"timeouts"`
-	Profiler          ServerProfiler          `config:"profiler"`
-	CompressionLevel  int                     `config:"compression_level"`
-	CompressionThresh int                     `config:"compression_threshold"`
-	Limits            ServerLimits            `config:"limits"`
-	Runtime           Runtime                 `config:"runtime"`
-	Bulk              ServerBulk              `config:"bulk"`
-	GC                GC                      `config:"gc"`
-	Instrumentation   Instrumentation         `config:"instrumentation"`
+	Host               string                  `config:"host"`
+	Port               uint16                  `config:"port"`
+	InternalPort       uint16                  `config:"internal_port"`
+	TLS                *tlscommon.ServerConfig `config:"ssl"`
+	Timeouts           ServerTimeouts          `config:"timeouts"`
+	Profiler           ServerProfiler          `config:"profiler"`
+	CompressionLevel   int                     `config:"compression_level"`
+	CompressionThresh  int                     `config:"compression_threshold"`
+	Limits             ServerLimits            `config:"limits"`
+	Runtime            Runtime                 `config:"runtime"`
+	Bulk               ServerBulk              `config:"bulk"`
+	GC                 GC                      `config:"gc"`
+	Instrumentation    Instrumentation         `config:"instrumentation"`
+	StaticPolicyTokens StaticPolicyTokens      `config:"static_policy_tokens"`
```

Comment:
>I think that having enrollment tokens as a list as @joshdover suggested is the most straightforward.
And we should add a description in fleet-server.reference.yml.

Thank you for the feedback, added the reference to `fleet-server.reference.yml`

>I don't think we need to be concerned with list/map performance implications on our initial implementation, it can be changed later if we need to.

I dont think that it would be that easy to change in the future but I have updated it to hold a slice.